adapters/connectkit-connector/src/bloctoConnector.ts

```diff
@@ -0,0 +1,232 @@
+import {
```

Comment:
should use `@blocto/wagmi-connector` and override some function ?

Reply:
I anticipate solving this issue in the next enhancement version. Until then, I need more testing to ensure the compatibility of wagmi-connector.